tests/draw_test.py

```diff
@@ -31,8 +31,13 @@ class TestBase(unittest.TestCase):
 
     def setUp(self):
         tests_dir = os.path.dirname(os.path.realpath(__file__))
-        self.tests_data_dir = os.path.abspath(os.path.join(tests_dir, './data'))
-        self.tests_blessed_images_dir = os.path.abspath(os.path.join(tests_dir, './blessed_images'))
+        self.tests_data_dir = os.path.abspath(os.path.join(
+            tests_dir, "../../worldengine-data/tests/data"))
+        self.tests_blessed_images_dir = os.path.abspath(
```

Comment:
blessed images are regenerated by the data_generator.py file under tests/data. If we move the blessed images we could move also that file